modules/nf-core/propr/propr/templates/propr.R

```diff
@@ -96,6 +96,43 @@ set_reference <- function(ivar, mat){
     return(ivar)
 }
 
+#' Extract the proportionality cutoff for a specified FDR value.
+#' Gene pairs with a proportionality value higher than the extracted cutoff will be considered significantly proportional.
+#'
+#' @param object propr object. Output from propr function. updateCutoffs function should be applied to the object previous to valCutoff.
+#' @param fdrVal FDR value to extract the cutoff for. Per default 0.05
+#'
+#' @return cutoff value. Proportionality values higher than this cutoff are considered significant.
+valCutoff  <- function(object, fdrVal = 0.05){
+    fdr_df <- object@fdr
+    if (prod(dim(fdr_df) == 0)){
+        warning("Please run updateCutoff on propr first")
+    }else{
+        fdr_vals <- fdr_df\$FDR
+        threshold <- any(fdr_vals <= fdrVal)
+        if (threshold == TRUE){
+            fdr_threshold <- fdr_vals[which.max(fdr_vals <= fdrVal)]
+            print(fdr_threshold)
+        }else{
+            warning("FDR is higher than the specified threshold for all proportionality values. Using the lowest fdr instead")
+            fdr_threshold <- fdr_vals[length(fdr_vals)]
+        }
+    cutoff <- fdr_df\$cutoff[fdr_df\$FDR == fdr_threshold]
+    }
+    return(cutoff)
+}
+
+#' Convert a proportionality matrix to an adjacency matrix based on a threshold.
+#'
+#' @param matrix proportionality matrix. Can be extracted from propr object with getMatrix().
+#' @param cutoff Significant proportionality value extracted from valCutoff function.
+#'
+#' @return Adjacency matrix. Gene pairs with a proportionality value higher than the threshold will have 1, otherwise 0.
+convert_to_adjacency <- function(matrix, cutoff) {
```

Comment:
some metrics like 'rho' and 'pcor' they are better with high values. But other metrics like 'phi' or 'phs' they are the contrary.
You should consider this when converting to adjacency matrix.

Reply:
I have modified the convert_to_adjacency and valCutoff function to include this